src/test/java/com/microfocus/application/automation/tools/results/service/TestDefaultExternalEntityUploadServiceImpl.java

```diff
@@ -36,10 +54,13 @@ private static void testJunit(int i) throws Exception{
 				"",
 				""
 				);
+		int i = 107;
```

Comment:
What is the use of this variable across all tests?

Reply:
I don't know. I didn't write these test. I think they are for making some difference.